tests/tests_integration/test_api/test_sequences_data.py

```diff
@@ -183,7 +183,8 @@ def test_delete_range(self, cognite_client, new_seq_long):
             column_external_ids=new_seq_long.column_external_ids, rows=data, id=new_seq_long.id
         )
         cognite_client.sequences.data.delete_range(start=4, end=15, id=new_seq_long.id)
-        dps = cognite_client.sequences.data.retrieve(start=0, end=None, id=new_seq_long.id)
+
         # potential delay, so can't assert, but tested in notebook
+        # dps = cognite_client.sequences.data.retrieve(start=0, end=None, id=new_seq_long.id)
```

Comment:
mistaken?

Reply:
Nope. The result isn't used for anything - see the commented out assertions below, just commenting out the rest so mypy doesn't complain about unused variables.